### PR TITLE
Descriptions

### PR DESCRIPTION
**Pull request by michalbak1212**
Inroducing file, concerning all of the main issues, connected with implementation of the library.

### CODE REVIEW

File: Descriptions.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+![Jimage Logo](https://github.com/kmisztal/JImageStreamToolkit/blob/master/Jimage_logo.png)
+
+
+
+Jest ogólnodostępnym, otwartym na rozszerzanie, pakietem, pozwalającym na obróbkę zdjęć.
```
Comment (slaby93) [inline Descriptions.md]: Only English

Comment (slaby93) [inline Descriptions.md]: Only English in did

**Comment by slaby93** [comment]
Where is progress?

File: Descriptions.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+![Jimage Logo](https://github.com/kmisztal/JImageStreamToolkit/blob/master/Jimage_logo.png)
+
+Is an open package, enabling photos customisation.
```
Comment (slaby93) [inline Descriptions.md]: photos -> image

**Comment by michalbak1212** [comment]
Updated.

**Pull request closed without merging**